tests/Api/Response/ResponseParserTest.php

```diff
@@ -39,12 +39,19 @@ public function setUp()
         $this->loggerFactory->expects($this->any())->method('getLogger')->willReturn($this->logger);
     }
 
+    /**
+     * @runInSeparateProcess
```

Comment:
Why is this necessary?

Reply:
It fails the test without it. The problem is with location header being added. 
Found a fix with process isolation here: https://github.com/sebastianbergmann/phpunit/issues/720